### ISSUE TITLE
Validation is only triggered on selecting another control

### ISSUE THREAD

**Issue by robshaffbbc** [orbeon/orbeon-forms#1503]
To recreate:
1. Create a new form
2. The new form opens with one text control by default. Go to the validation tab for this control.
3. Make into a mandatory integer field.
4. Add another blank text control
5. Save, publish and open form runner for the new form.
6. Click in the field with validation.
7. Change focus away from the control with validation (but not into the other control).
-It will not give an error to indicate that the field is mandatory.
8. Click in the field with validation
Enter a text value and change focus away from the control with validation (but not into the other control)
-It will not give an error to indicate that the field needs to be an integer
1. Click back into the field with validation and then out to the other control.
   Validation will be triggered and an error message will display for both cases.
2. Now click into the validated field and enter text data and change focus away from the control with validation (but not into the other control). 
   -An error will now display.

In summary: validation will only be triggered when clicking into a control and then into another control. Validation is thereafter triggered even when not clicking into another control.
